d8/d8batch/config/install/field.storage.node.field_feed_source_url.yml

```diff
@@ -0,0 +1,19 @@
+uuid: 6d09794e-a019-42ae-9a43-67df9009c074
+langcode: en
```

Comment:
Why does a configuration (entity?) need a language key if it doesn't have a label/is not translatable?